sketch/sketch.c

```diff
@@ -115,10 +117,13 @@ void display_shapes()
 void draw_grid()
 {
     set_stroke("gray");
-    for (int i = -300; i < 300; i += 100)
+    for (int i = -sketch_height / 2; i < sketch_height / 2; i += 100)
     {
-        draw_line(i, -300, i, 300);
-        draw_line(-300, i, 300, i);
+        draw_line(-sketch_width, i, sketch_width, i);
+    }
+    for (int j = -sketch_width; j < sketch_width; j += 100)
```

Comment:
Shouldn't this be `-sketch_width/2`?

Reply:
Yes I will change it and commit thank you